chart/templates/mayastor/rbac/rbac.yaml

```
@@ -32,6 +32,14 @@ rules:
 - apiGroups: ["openebs.io"]
   resources: ["diskpools/status"]
   verbs: ["update", "patch"]
+  # must read cm info
+- apiGroups: [""]
+  resources: ["configmaps"]
+  verbs: ["get", "update", "patch"]
```

Now that we are creating the configmap using code.
```suggestion
  verbs: ["create", "get", "update", "patch"]
```